### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by RachelAnnB**
-fixed footer hovers
-changed header at mobile size
-compressed all images
-other small code tweaks

### CODE REVIEW

File: src/scss/_faq.scss [hunk 1]
```diff
@@ -78,30 +96,38 @@ $bp-faq-small: 50em; //800px
...
+
+    &__box {
+      width: 95%;
+      display: none;
+      padding: 0.25em; //4px
+      margin: auto;
+    }
+
+    &__box.show {
+      display: block !important;
```
Comment (asimpson) [inline src/scss/_faq.scss:117]: I don't _think_ you need the `important` here?

File: src/scss/_footer.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
 //---------------Break Points---------------//
```
Comment (asimpson) [inline src/scss/_footer.scss:1]: whoa, this file is now spaced 4 instead of 2? 🤔

File: src/scss/_header.scss [hunk 3]
```diff
@@ -1,22 +1,31 @@
...
-}
-
 .site-title {
 	padding-top: 0.5em; //8px
 	margin: auto;
 
+	@media(max-width: $bp-header-small) {
+		width: 30%;
+		float: left;
+		position: absolute;
```
Comment (asimpson) [inline src/scss/_header.scss:24]: I _think_ you want this to be in applied later?

**Comment by asimpson** [comment]
This is looking great! I made a few nit-picky comments, but it's coming along nicely 👍 💯

**Comment by RachelAnnB** [comment]
Thanks Adam!

File: src/scss/_header.scss (hunk 3 above)
Comment (RachelAnnB) [reply src/scss/_header.scss:24]: What do you mean by this?

**Pull request merged**